File: src/Obstaculo.java
public class Obstaculo extends Jugador {

    public Obstaculo(int fila, int columna) {
        super(0, 0, fila, columna, "X"); 
        this.estaVivo = false; 
    }

    @Override
    public void moverJugador(int f, int c) {
        
    }
}

File: obstaculo.js
/**
 * Clase Obstaculo que extiende Jugador
 */
class Obstaculo extends Jugador {
    constructor(fila, columna) {
        // Roguelike clásico: muro/obstáculo como '#'
        super(fila, columna, "#");
    }
}
